MAUI/DataForm/migration.md

```diff
@@ -0,0 +1,1085 @@
+</tr>
+
+<tr>
+<td>{{'[ValueMemberPath](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormPickerItem.html#Syncfusion_XForms_DataForm_DataFormPickerItem_ValueMemberPath)'| markdownify }}</td>
+<td>SelectedValuePath</td>
+<td>Gets or sets a path to a value on the source object which will get saved to data field.</td>
+</tr>
+</table>
+
+### DataFormComboBoxItem
+
+<table>
+<tr>
+<th>Xamarin SfDataForm</th>
+<th>.NET MAUI SfDataForm</th>
+<th>Description</th></tr>
+
+<tr>
+<td>Nil</td>
+<td>IsEditable</td>
+<td>Gets or sets a value indicating whether the ComboBox can be edited.</td>
+</tr>
+
+<tr>
+<td>Nil</td>
+<td>TextSearchMode</td>
+<td>Gets or sets the text search mode for ComboBox editor.</td>
+</tr>
+</table>
+
+### DataFormAutoCompleteItem
+
+<table>
+
+<tr>
+<th>Xamarin SfDataForm</th>
+<th>.NET MAUI SfDataForm</th>
+<th>Description</th></tr>
+
+<tr>
+<td>{{'[AutoCompleteMode](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormAutoCompleteItem.html#Syncfusion_XForms_DataForm_DataFormAutoCompleteItem_AutoCompleteMode)'| markdownify }}</td>
+<td>TextSearchMode</td>
+<td>Gets or sets the text search mode for auto complete editor.</td>
+</tr>
+</table>
+
+### DataFormDisplayOptionsAttribute
+
+<table>
+
+<tr>
+<th>Xamarin SfDataForm</th>
+<th>.NET MAUI SfDataForm</th>
+<th>Description</th></tr>
+
+<tr>
+<td>{{'[ColumnSpan](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DisplayOptionsAttribute.html#Syncfusion_XForms_DataForm_DisplayOptionsAttribute_ColumnSpan)'| markdownify }}</td>
+<td>ColumnSpan</td>
+<td>Gets or sets the number of column that this item spans to increase item's width.</td>
+</tr>
+
+<tr>
+<td>{{'[RowSpan](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DisplayOptionsAttribute.html#Syncfusion_XForms_DataForm_DisplayOptionsAttribute_RowSpan)'| markdownify }}</td>
+<td>RowSpan</td>
+<td>Gets or sets the number of rows that this item spans which is used to increase item's height.</td>
+</tr>
+
+<tr>
+<td>Nil</td>
+<td>RowOrder</td>
+<td>Gets or sets the row order of the editor in the data form.</td>
+</tr>
+
+<tr>
+<td>Nil</td>
+<td>ItemsOrderInRow</td>
+<td>Gets or sets the editor’s position in a row.</td>
+</tr>
+
+<tr>
+<td>{{'[ShowLabel](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DisplayOptionsAttribute.html#Syncfusion_XForms_DataForm_DisplayOptionsAttribute_ShowLabel)'| markdownify }}</td>
+<td>ShowLabel</td>
+<td>Gets or sets a value indicating whether to show the label to the DataForm editor.</td>
+</tr>
+
+<tr>
+<td>{{'[ValidMessage](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DisplayOptionsAttribute.html#Syncfusion_XForms_DataForm_DisplayOptionsAttribute_ValidMessage)'| markdownify }}</td>
+<td>ValidMessage</td>
+<td>Gets or sets a string which will get displayed when user enters valid value for the data field as success message.</td>
```

Comment:
<td>Gets or sets a string displayed when the user enters a valid value for the data field as a success message.</td>